packages/berry-pnpify/sources/NodeModulesFS.ts

```diff
@@ -135,11 +161,17 @@ export class NodeModulesFS extends FakeFS {
   }
 
   async lstatPromise(p: string) {
-    return await this.baseFs.lstatPromise(this.resolveDirOrFilePath(p));
+    return this.resolveLink(p, 'lstat',
+      (stats) => Object.assign(stats, { isSymbolicLink: () => true }),
```

Comment:
This won't prevent `isFile` & co to return `true` as well. Maybe abstract this in a `makeSymlinkStats` helper function?

Reply:
Yep, thanks, good catch